.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: go
+
+go:
+  - 1.4.2
+  - 1.4.1
+  - 1.3.3
```

Comment:
Indeed, `1.4` appears to be exactly `1.4`. It's Ruby tools that interpret short versions to mean "latest `1.4.x`".

> define the versions it's actually expected to build and pass on

As a public project, I would think we'd want to support as many versions as possible. For example, [trusty ships with 1.2.1](http://packages.ubuntu.com/trusty/golang). If it is inexpensive to verify support for older versions, I think we should do it.

We can still declare any version unsupported whenever we want.